### DISCUSSION TITLE
30 Days of PWA blog series

### DISCUSSION THREAD

**Discussion by beth-panx** [microsoft/win-student-devs#24] [General]
This is the discussion area for blog series 30 Days of PWA. Please feel free to leave comments, ask questions, and provide us your feedback!

**Comment by meotimdihia** [comment]
PWA is good but it is buggy on iOS. This is a video recording that my user reported:

"this is an ipad bug that when you left the pwa app and comeback, it will automatically zoom in".
https://www.youtube.com/shorts/PY5208yCbxM 

And a lot of other bugs too and missing features vs native apps.
